src/promptflow/tests/sdk_cli_azure_test/unittests/test_azure_cli_perf.py

```diff
@@ -56,10 +38,16 @@ def operation_scope_args(subscription_id: str, resource_group_name: str, workspa
     ]
 
 
-@pytest.mark.usefixtures("mock_get_azure_pf_client")
 @pytest.mark.perf_monitor_test
-class TestAzureCliTimeConsume:
-    def test_pfazure_run_create(self, operation_scope_args, time_limit=30):
+@pytest.mark.usefixtures(
+    "mock_get_azure_pf_client",
+    "mock_set_headers_with_user_aml_token",
+    "single_worker_thread_pool",
+    "vcr_recording",
+)
+class TestAzureCliPerf:
+    def test_pfazure_run_create(self, operation_scope_args, runtime: str, randstr: Callable[[str], str], time_limit=30):
```

Comment:
Local testing for 0.4s.
Due to the dashboard displaying a maximum value of 20 seconds, the timelimit has not been modified temporarily. It will be modified after it stabilizes in the future.